requirements/dev.txt

```diff
@@ -44,7 +45,7 @@ pytz==2017.2              # via django
 requests-oauthlib==0.8.0  # via social-auth-core
 requests==2.18.4          # via requests-oauthlib, social-auth-core
 simplegeneric==0.8.1      # via ipython
-six==1.10.0               # via pip-tools, prompt-toolkit, social-auth-app-django, social-auth-core, traitlets
+six==1.10.0               # via model-mommy, pip-tools, prompt-toolkit, social-auth-app-django, social-auth-core, traitlets
```

Comment:
Where do these `via`s come from? Do we add them manually or are they generated somehow?

Reply:
They're generated by pip-tools compiling the `dev.in` file.

Reply:
Incidentally, it seems the new `dev.in` has been missed from the PR!

Reply:
It shouldn't be needed as nothing has changed directly in the dev dependencies. `dev.in` includes `test.in` which did change, which is why the resulting `dev.txt` has changed.

Reply:
Ah, I see - sorry!